__tests__/app.test.js

```diff
@@ -108,3 +106,55 @@ describe("GET API/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("should an array of comments for the given review_id of which each comment should have the given properties", () => {
+    const reviewId = 2;
+    return request(app)
+      .get(`/api/reviews/${reviewId}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Array);
+        body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("review_id");
+        });
+        expect(body.comments).toBeSorted("created_at", { descending: true });
```

Comment:
could this be 2 tests, one to check the properties of the comments and one to test the order is returned correctly